uMenu/source/ui/ui_QuickMenu.cpp

```diff
@@ -8,6 +8,10 @@ extern ui::MenuApplication::Ref g_MenuApplication;
 
 namespace ui {
 
+    namespace quickmenu_utils{
```

Comment:
As mentioned in another comment, I'd make this an anonymous namespace (also keeping the name style for global variables) and modify it with helper functions rather than using externs